tests/network/dual_stack_cluster.go

```diff
@@ -0,0 +1,32 @@
+package network
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"kubevirt.io/client-go/kubecli"
+	"kubevirt.io/kubevirt/tests/flags"
+	"kubevirt.io/kubevirt/tests/libnet"
+)
+
+var _ = SIGDescribe("Dual stack cluster network configuration", func() {
+	var virtClient kubecli.KubevirtClient
+
+	BeforeEach(func() {
+		var err error
+		virtClient, err = kubecli.GetKubevirtClient()
+		Expect(err).NotTo(HaveOccurred(), "Should successfully initialize an API client")
+	})
+
+	Context("when dual stack cluster configuration is requested", func() {
+		It("should report at least one IPv6 address on a KubeVirt system pod", func() {
```

Comment:
This is describing something that happens only internally in one of the helpers. As silly as it sounds: When dual stack cluster testing is enabled, `Specify` the cluster should be dual-stack

Reply:
Done.